tests/devices/default_qubit/test_default_qubit_native_mcm.py

```diff
@@ -232,8 +259,7 @@ def func(x, y):
     results1 = func1(*params)
     results2 = func2(*params)
 
-    if postselect is None or measure_f in (qml.expval, qml.probs, qml.var):
-        validate_measurements(measure_f, shots, results1, results2)
+    validate_measurements(measure_f, shots, results1, results2)
```

Comment:
Sorry if I'm confused... Why is this possible now?

Reply:
I think it was always possible. My guess is that this was just a forgotten item to fix.